lib/shared/addon/components/cluster-driver/driver-okecapi/template.hbs

```diff
@@ -215,6 +215,134 @@
         }}
       </div>
     </div>
+    {{#accordion-list-item title=(t "clusterNew.ociocne.workers.title")
+                           detail=(t "clusterNew.ociocne.workers.detail")
+                           showExpand=false
+                           expandOnInit=true
+                           expandAll=al.expandAll
+                           expand=(action expandFn)
+    }}
+      <div class="row">
+        <button
+          class="btn btn-sm bg-primary inline-block mb-5"
+          style="margin-top: -5px;" type="button" {{action "addNodePool"}}
+        >
+          {{t "clusterNew.ociocne.nodePool.addNew"}}
+        </button>
+      </div>
+      {{#each nodePools as |np|}}
+        {{#accordion-list-item title=np.name
+                               detail="configure node pool"
+                               expandAll=expandAll
+                               expand=(action expandFn)
+                               expandOnInit=false
+        }}
+          <div class="row">
+            <div class="col span-4">
+              <label class="acc-label">
+                {{t 'clusterNew.ociocne.nodeName.label'}}{{field-required}}
+              </label>
+              {{#input-or-display editable=(eq step 2) value=np.name}}
+                {{input type="text" name="np-name" classNames="form-control" value=np.name}}
+                <p class="help-block">
+                  {{t 'clusterNew.ociocne.nodeName.help'}}
+                </p>
+              {{/input-or-display}}
+            </div>
+          </div>
+          <div class="row">
+            <div class="col span-4 mb-0">
+              <label class="acc-label">
+                {{t 'clusterNew.okecapi.k8sversion.label'}}
+              </label>
+              {{#input-or-display editable=(and (eq step 3) isNew) value=np.memory}}
```

Comment:
config.kubernetesVersion

Reply:
also, eq step 2

Reply:
shouldn't this also be np.kubernetesVersion?